packages/build/tests/install/snapshots/tests.js.md

```diff
@@ -44,6 +44,7 @@ Generated by [AVA](https://ava.li).
     Packaging Functions from functions directory:␊
      - index.js␊
     ␊
+    ␊
```

Comment:
You'll see this new line in a lot of snapshots. This is because we started adding a new line before the esbuild output when we first introduced it, but up until now we were able to print it only if the feature flag was enabled, so there was no need to update the snapshots for non-esbuild cases. That is not possible now, since the decision of whether or not to use esbuild will come from the configuration property for each function, so we have no option but to print the empty line regardless of what bundler is being used.